cumulusci/tests/pytest_plugins/pytest_sf_vcr.py

```diff
@@ -7,36 +7,117 @@
 """
 
 import re
+from contextlib import contextmanager
+from contextvars import ContextVar
+from pathlib import Path
 
-from .pytest_sf_orgconnect import sf_pytest_orgname
+import pytest
+from vcr import cassette
 
 
 def simplify_body(request_or_response_body):
     decoded = request_or_response_body.decode("utf-8")
-    if "<sessionId>" in decoded:
-        decoded = re.sub(
-            r"<sessionId>.*</sessionId>",
-            "<sessionId>**Elided**</sessionId>",
-            decoded,
-        )
-    decoded = re.sub(r"001[\w\d]{15,18}", "001ACCOUNTID", decoded)
+    decoded = _cleanup(decoded)
+    # if "GeocodeAccuracy" in decoded:
+    #     breakpoint()
+
     return decoded.encode()
 
 
-def sf_before_record_request(request):
-    if request.body:
-        request.body = simplify_body(request.body)
-    request.uri = re.sub(
-        r"//.*.my.salesforce.com", "//orgname.my.salesforce.com", request.uri
-    )
-    request.uri = re.sub(
-        r"//.*\d+.*.salesforce.com/", "//orgname.my.salesforce.com/", request.uri
-    )
-    request.uri = re.sub(r"00D[\w\d]{15,18}", "Organization/ORGID", request.uri)
+replacements = [
+    (r"/v?\d\d.0/", r"/vxx.0/"),
+    (r"/00D[\w\d]{12,15}", "/00D0xORGID00000000"),
+    (r'"00D[\w\d]{12,15}"', '"00D0xORGID00000000"'),
+    (r".com//", r".com/"),
+    (r"ersion>\d\d.0<", r"ersion>vxx.0<"),
+    (r"<sessionId>.*</sessionId>", r"<sessionId>**Elided**</sessionId>"),
+    (r"001[\w\d]{12,15}", "0010xACCOUNTID0000"),
+    (r"//.*.my.salesforce.com", "//orgname.my.salesforce.com"),
+    (r"//.*\d+.*.salesforce.com/", "//orgname.my.salesforce.com/"),
+    (
+        r"202\d-\d\d-\d\dT\d\d:\d\d\:\d\d.\d\d\d\+0000",
+        "2021-01-01T01:02:01.000+0000",
```

Comment:
Same concern as with Ids. Some tests may be reliant upon dates actually being different.